recipes/expat/all/conanfile.py

```diff
@@ -52,11 +51,15 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
         self.cpp_info.name = "EXPAT"
-        if self.settings.os == "Windows" and self.settings.build_type == "Debug":
-            self.cpp_info.libs = ["expatd"]
+        if self.settings.os == "Windows":
+            prefix = "lib" if self.version >= "2.2.9" else ""
```

Comment:
I hope this compares version as version, not as a string. @uilianries please confirm

Reply:
that's how I interpreted https://github.com/conan-io/conan/blob/develop/conans/client/tools/version.py#L48

Reply:
not sure if that `self.version` is a Version class

Reply:
IMO I prefer splitting or using `tools.collect_libs`. Generic recipes should not contain conditionals for specific versions.